core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -6294,6 +6294,21 @@ components:
           items:
             type: string
           description: the roles the caller has on the project
+        workspacesWithIncorrectBillingAccount:
```

Comment:
So we aren't also adding `workspacesWithCorrectBillingAccount` because then we'd need to populate it? Will adding it via the PPW feature branch later make things difficult for AoU?